### PR TITLE
PBM-1419 Reworked supported versions matrix

### PR DESCRIPTION
**Pull request by nastena1606**
Split into separate ones for MongoDB and PSMDB
Added version 2.7.0 to each

modified:   docs/details/versions.md

### CODE REVIEW

File: docs/details/versions.md [hunk 1]
```diff
@@ -2,26 +2,41 @@
 
 Percona Backup for MongoDB is compatible with:
 
-* MongoDB Community / Enterprise Edition  with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) enabled for logical backups.
-* Percona Server for MongoDB with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) for logical backups. For physical backups, also configure WiredTiger as the storage engine.
+* MongoDB Community / Enterprise Edition with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) enabled. Percona Backup for MongoDB supports only *logical backups*.
+* Percona Server for MongoDB with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) for logical backups. For physical and incremental backups, also configure WiredTiger as the storage engine.
 
-The following table provides information about the supported MongoDB versions for each PBM release. 
+Navigate to the required compatibility matrix to see what MongoDB versions your current PBM release is compatible with for backups and restores.
```
Comment (radoslawszulgo) [inline docs/details/versions.md:8]:
```suggestion
Refer to the compatibility matrices below to check which MongoDB versions your current PBM release supports for backups and restores.
```

File: docs/details/versions.md [hunk 2]
```diff
@@ -2,26 +2,41 @@
...
-| [2.1.0](../release-notes/2.1.0.md) | <ul><li>Logical backups: version 4.4 and higher</li></ul> | <ul><li>Logical backups: version 4.4 and higher</li><li>Physical backups - version 4.4.6-8, 5.0.x, 6.0.x</li><li>Incremental backups: version 4.2.24-24, 4.4.18, 5.0.14-12, 6.0.3-2 and higher</li></ul>| No. A fresh backup is required|
-| [1.7.0](../release-notes/1.7.0.md) | <ul><li>Logical backups: version 4.2 and higher</li></ul> | <ul><li>Logical backups: version 4.2 and higher</li><li>Physical backups (tech preview): version 4.2.15-16, 4.4.6-8, 5.0 and higher</li></ul> | Yes
-| [1.6.1](../release-notes/1.6.1.md) | <ul><li>Logical backups: version 3.6 and higher with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) enabled</li></ul> | <ul><li>Logical backups: version 3.6 and higher</li></ul>|Yes
+The following table lists the Percona Server for MongoDB versions supported for each backup type.
```
Comment (radoslawszulgo) [inline docs/details/versions.md:16]: We can link Percona Server for MongoDB to PSMDB docs

File: docs/details/versions.md [hunk 3]
```diff
@@ -2,26 +2,41 @@
 
 Percona Backup for MongoDB is compatible with:
 
-* MongoDB Community / Enterprise Edition  with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) enabled for logical backups.
-* Percona Server for MongoDB with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) for logical backups. For physical backups, also configure WiredTiger as the storage engine.
+* MongoDB Community / Enterprise Edition with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) enabled. Percona Backup for MongoDB supports only *logical backups*.
+* Percona Server for MongoDB with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) for logical backups. For physical and incremental backups, also configure WiredTiger as the storage engine.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/details/versions.md:6]: Reworked. PSMDB also supports Memory Engine, so I changed the wording to ensure WT is set

File: docs/details/versions.md [hunk 4]
```diff
@@ -2,26 +2,41 @@
...
 
-The following table provides information about the supported MongoDB versions for each PBM release. 
+Navigate to the required compatibility matrix to see what MongoDB versions your current PBM release is compatible with for backups and restores. 
 
 !!! note ""
 
-    End-of-life MongoDB versions may work with PBM, but they are not tested for compatibility. Consider using an previous PBM version in this case.
+    End-of-life MongoDB versions may work with PBM, but they are not tested for compatibility. Consider using a previous PBM version in this case.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/details/versions.md:12]: Reworded

File: docs/details/versions.md (hunk 2 above)
Comment (nastena1606) [reply docs/details/versions.md:16]: Done

File: docs/details/versions.md [hunk 5]
```diff
@@ -2,26 +2,41 @@
...
-| [2.1.0](../release-notes/2.1.0.md) | <ul><li>Logical backups: version 4.4 and higher</li></ul> | <ul><li>Logical backups: version 4.4 and higher</li><li>Physical backups - version 4.4.6-8, 5.0.x, 6.0.x</li><li>Incremental backups: version 4.2.24-24, 4.4.18, 5.0.14-12, 6.0.3-2 and higher</li></ul>| No. A fresh backup is required|
-| [1.7.0](../release-notes/1.7.0.md) | <ul><li>Logical backups: version 4.2 and higher</li></ul> | <ul><li>Logical backups: version 4.2 and higher</li><li>Physical backups (tech preview): version 4.2.15-16, 4.4.6-8, 5.0 and higher</li></ul> | Yes
-| [1.6.1](../release-notes/1.6.1.md) | <ul><li>Logical backups: version 3.6 and higher with [MongoDB Replication :octicons-link-external-16:](https://docs.mongodb.com/manual/replication/) enabled</li></ul> | <ul><li>Logical backups: version 3.6 and higher</li></ul>|Yes
+The following table lists the Percona Server for MongoDB versions supported for each backup type.
+
+Each entry indicates the PBM version that introduces changes in the supported Percona Server for MongoDB versions.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/details/versions.md:18]: Added

**Review by igroene — APPROVED** [review]
LGTM

**Review by radoslawszulgo — APPROVED** [review]

**Pull request merged**